### ISSUE TITLE
Error when save modifications from web editor

### ISSUE THREAD

**Issue by Teyk0o** [LuckPerms/LuckPerms#3967]
### Description

I do some modifications in my web editor and when I hit the "Apply" button, an error appears in my server console.
After some seconds (30/40) I can apply my modifications with ``/lp applyedits`` on my server.

### Reproduction Steps

- Open the web editor with ``/lp editor`` command on your server
- Do somes modifications on the editor
- Click on the ``Apply`` button
- See the error in the server console

### Expected Behaviour

No error

### Server Details

Fabric 1.20.1 Loader 0.16.2 Launcher 1.0.1

### LuckPerms Version

v5.4.102

### Logs and Configs

https://gist.github.com/Teyk0o/469a9a307a87013bd15071dc8e5f99dd

**Comment by BNTFryingPan** [comment]
this just seems like a network issue. are you still having this problem?

**Comment by Laarryy** [comment]
Yep - standard network timeout, which LP can't anticipate or correct for. Closing as this isn't a bug.
